bool approach_traj_finished

bool traj_finished
